dist/linkjuice.js

```diff
@@ -1,51 +1,44 @@
-/*! linkjuice.js v1.0.0 | (c) 2016 @toddmotto | https://github.com/toddmotto/linkjuice */
-(function (global, factory) {
-  if (typeof define === "function" && define.amd) {
```

Comment:
No this all is caused by removing `.babelrc`, which was caused because babel was unable to find the specified plugin in it, and which inturn was due to that package was not listed in `package.json` and hence did not came with `npm i` So, my recent commits should fix this.